src/main.c

```diff
@@ -28,21 +28,25 @@
 
 static void print_usage()
 {
-    printf("Usage: -h host [-p port] -t topic1 [-t topicN]\n");
+    printf("Usage: -h host [-p port] [-u username] [-P password] -t topic1 [-t topicN]\n");
     printf(" -h: MQTT host to connect to\n");
     printf(" -p: MQTT port to connect to (1883)\n");
+    printf(" -u: MQTT username\n");
+    printf(" -P: MQTT password\n");    
```

Comment:
You have trailing spaces here. Should be easy to spot when you run git diff before or git show after commit.